src/main/scala/cromwell/binding/types/WdlAnyType.scala

```diff
@@ -0,0 +1,13 @@
+package cromwell.binding.types
+
+case object WdlAnyType extends WdlType {
+  val toWdlString: String = s"Any"
+
+  override protected def coercion = {
+    case any: Any =>
+      WdlType.wdlTypes.map(_.coerceRawValue(any)).find(_.isSuccess) match {
+        case Some(value) => value.get
+        case None => throw new UnsupportedOperationException(s"Could not coerce $any into a WDL type")
```

Comment:
This is literally a `getOrElse` although throwing exceptions in there admittedly can lead to ridiculously long lines